scripts/run_driver_tests.psm1

```diff
@@ -460,7 +459,7 @@ function Invoke-ConnectRedirectTest
 function Invoke-CICDStressTests
 {
     param([parameter(Mandatory = $true)][bool] $VerboseLogs,
-          [parameter(Mandatory = $false)][int] $TestHangTimeout = 3600,
+          [parameter(Mandatory = $false)][int] $TestHangTimeout = (45*60),
```

Comment:
The stress tests were NOT passing any timeout value, which would mean a default of 10 minutes. However, there are multiple stress test cases, which each utilize the -td=5 (5 minutes) test parameter. So surely this is insufficient.

This looks like it might be the RCA of the km_stress_test failures, but NOT the RCA of the km_stress_test with restart extension failures.